airflow/config_templates/default_celery.py

```diff
@@ -23,13 +23,27 @@
 from airflow.exceptions import AirflowConfigException, AirflowException
 from airflow.utils.log.logging_mixin import LoggingMixin
 
+
+def _broker_supports_visibility_timeout(url):
+    # Only certain transports support the 'visibility_timeout' option.
+    #
+    # See:
+    # http://docs.celeryproject.org/en/master/userguide/configuration.html#broker-transport-options
+    if url.startswith("redis") or url.startswith("sqs"):
```

Comment:
Should probably be `redis://` etc. Otherwise this looks right.

Does SQLA broker work now with this change? i.e. is it enough?

Reply:
Simpler version:

```
def _broker_supports_visibility_timeout(url):
    return url.startswith("redis://") or url.startswith("sqs://")
```

Reply:
@ashb I have incorporated your change